src/Microsoft.PowerShell.ConsoleHost/host/msh/ProgressPane.cs

```diff
@@ -301,7 +301,15 @@ private void WriteContent()
         {
             if (_content is not null)
             {
-                Console.CursorVisible = false;
+                try
+                {
+                    Console.CursorVisible = false;
+                }
+                catch (System.IO.IOException)
+                {
+                    // ignore if set can't set cursor visibility
+                }
+
```

Comment:
I'm able to reproduce the issue by redirecting the standard out of the pwsh process:

![image](https://user-images.githubusercontent.com/127450/107101972-d2f28d80-67cd-11eb-89e4-f4c420917dc5.png)

Maybe the right fix would be to not render the progress when the standard output of the current process is redirected? Or revert back to the `Classic` view when stdout is redirected? (I personally think the former makes more sense).

Reply:
Yeah, that would make sense to not render progress at all.

Reply:
I refactored the `WriteProgress` method body a little bit to reduce nesting there. It's easier to see the difference by ignoring the whitespaces: https://github.com/PowerShell/PowerShell/pull/14716/files?w=1

Reply:
LGTM